docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,204 @@
+
+### Start Usage
+
+```sh
+SOME_ENV_VAR=value npm run start -- <flags>
+# e.g. ONE_CONFIG_ENV=development npm run start -- --root-module-name=my-root-module
+```
+
+#### Flags
+
+| Flag       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `--root-module-name`  | Name of the [Holocron Module] that serves as the entry point to your application. |
+| `--module-map-url`  | Remote Module Map URL for [`one-app-dev-cdn`] to proxy. *For Development.* |
+| `--use-middleware` | Apply a custom middleware configuration for [`one-app-dev-proxy`]. *For Development.* |
+
+#### Environment Variables
+
+Please see the [Environment Variables](./Environment-Variables.md) API docs.
+
+## Test Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `test` | Runs all `test:*` scripts. |
+| `test:unit` | Runs all [Jest] Unit Tests. Accepts Jest flags. |
+| `test:lint` | Runs [ESLint] [rules](https://github.com/americanexpress/eslint-config-amex) against source. |
+| `pretest:integration` | Using [`concurrently`], runs `build:prod-sample` and pulls Docker containers to prepare the [Integration Test Suite]. |
+| `test:integration` | Executes the [Integration Test Suite] using [Jest]. |
+| `test:danger` | Runs the [DangerJS tool](https://danger.systems/js/) that returns bundle size stats and a preflight checklist for PRs. |
+| `test:git-history` | Validates the Git commit message format using the [`commitlint`](https://github.com/conventional-changelog/commitlint) tool. |
+| `test:lockfile` | Validates that the `package-lock.json` file contains valid NPM registry URLs using the [`lockfile-lint`](https://github.com/lirantal/lockfile-lint) tool. |
+| `prestart:prod-sample` | Runs [`build:sample-modules`](#build-commands) before `start:prod-sample`. |
+| `start:prod-sample` | Runs a [Docker Compose](https://docs.docker.com/compose/compose-file/) file to start up the [Integration Test Suite] containers. |
+
+## Development Commands
+
+> 💬 These commands only work in Development
+
+| Flag | Description |
+|------|-------------|
+| `serve-module` | Accepts a path to a [Holocron Module] and adds it to the local development CDN. *See [`serve-module` Arguments](#serve-module-arguments).*  |
+| `drop-module` | Accepts a [Holocron Module] name and removes the Module from the local development CDN. *See [`drop-module` Arguments](#drop-module-arguments).* |
+| `set-middleware` | Accepts a Node JS file that defines custom [Express Middleware](http://expressjs.com/en/guide/writing-middleware.html#writing-middleware-for-use-in-express-apps) for the development proxy. *See [`set-middleware` Arguments](#set-middleware-arguments).* |
```

Comment:
> Accepts a Node JS file

The argument is a path to a file, not a file. I don't think its necessary to specify "Node"

Reply:
Fixed in a88378aafc426e868db005159f4e769323b6a93d